sycl/include/sycl/ext/xilinx/fpga/memory_properties.hpp

```diff
@@ -14,6 +14,10 @@
 
 __SYCL_INLINE_NAMESPACE(cl) {
 
+// TODO(lforg37): [Technical Debt]: Merge properties for hbm_bank and ddr_bank with a template type for the 
```

Comment:
Is it difficult to do?

Reply:
Difficult probably not so much, but of a long haul, as it requires modification in the property->annotation system.